styles/formSectionStyles.css

```diff
@@ -0,0 +1,83 @@
+* {
+    box-sizing: border-box;
+}
+
+.form__button {
+    background: linear-gradient(270deg, #FFC53D 14%, #FF9A3D 68.5%);
+    border: none;
+    border-radius: 15px;
+    color: #FFF;
+    cursor: pointer;
+    float: right;
+    font-family: "Raleway", sans-serif;
+    font-size: 18px;
+    font-weight: 700;
+    outline: none;
+    padding: 15px 27.5px;
+    transition: transform .2s ease-in-out;
+}
+
+.form__button:hover {
+    background: white;
+    border: 1px solid #FFC53D;
+    color: #FFC53D;
+    transform: scale(1.02);
+}
+
+.form div {
+    margin-bottom: 16px;
+}
+
+.form div:nth-child(4) {
+    margin-bottom: 32px;
+}
+
+.form__input[name="name"] {
+    box-shadow: inset 0px 4px 4px rgba(0, 0, 0, 0.15);
+}
```

Comment:
Taki box-shadow powinien być na elemencie w którym właśnie wprowadzamy dane albo na który najechaliśmy

Reply:
fakt, nie wpadłem na to :D zmieniam, dzięki!